wallet-new/src/Cardano/Wallet/API/V1/Accounts.hs

```diff
@@ -19,9 +17,8 @@ type API
     :<|> "accounts" :> WalletRequestParams
                     :> Summary "Retrieves the full list of Accounts."
                     :> Get '[JSON] (WalletResponse [Account])
-    :<|> "accounts" :> Header  "Daedalus-Passphrase" Text
-                    :> Summary "Creates a new Account for the given Wallet."
-                    :> ReqBody '[JSON] (New Account)
+    :<|> "accounts" :> Summary "Creates a new Account for the given Wallet."
+                    :> ReqBody '[JSON] NewAccount
```

Comment:
I think there is still value in leaving `(New Account)` there, because it means we can change the underlying type (if we ever want) without breaking the API type signature.